Tests/Engine/InvokeFormatter.tests.ps1

```diff
@@ -30,6 +30,15 @@ function foo {
 
             Invoke-Formatter $def $settings | Should -Be $expected
         }
+
+        It "Should not expand unary operators when being used as a single negative argument" {
+            $script = '$foo.bar(-$a)' 
+            Invoke-Formatter '$foo.bar(-$a)' -Settings CodeFormatting | Should -Be $script
+        }
+
+        It "Should expand unary operators when not being used as a single negative argument" {
```

Comment:
i'm not sure it's not just parens where this be applied. Take the following:
```
PS> Invoke-Formatter -ScriptDefinition '$r = $a -lt -$var'                                                                                                       
$r = $a -lt - $var
```
shouldn't that be `$r = $a -lt -$var`?

Reply:
I agree with you. I will extract this into a new issue. I am sure there are other special cases as well but I guess the best approach is to find them one by one and find heuristics to detect each case I suppose.